### PR TITLE
Several minor fixes

### PR DESCRIPTION
**Pull request by ofrei**
@MelLain Could you take a look? I'd like to merge this into stable branch, and forward-integrate to master.

### CODE REVIEW

File: src/artm/core/dictionary_operations.cc [hunk 1]
```diff
@@ -94,6 +94,10 @@ void DictionaryOperations::Export(const ExportDictionaryArgs& args, const Dictio
   }
 
   std::string str = token_dict_data.SerializeAsString();
+  if (str.size() >= 2147483647ULL)
+    BOOST_THROW_EXCEPTION(InvalidOperation("Dictionary " +
+      args.dictionary_name() + " is too large to export"));
```
Comment (MelLain) [inline src/artm/core/dictionary_operations.cc]: minor: for export

File: src/artm/core/dictionary_operations.cc [hunk 2]
```diff
@@ -94,6 +94,10 @@ void DictionaryOperations::Export(const ExportDictionaryArgs& args, const Dictio
   }
 
   std::string str = token_dict_data.SerializeAsString();
+  if (str.size() >= 2147483647ULL)
```
Comment (MelLain) [inline src/artm/core/dictionary_operations.cc]: Put this value into constant here or in common.h

File: src/artm/core/dictionary_operations.cc [hunk 3]
```diff
@@ -562,5 +571,18 @@ void DictionaryOperations::StoreIntoDictionaryData(const Dictionary& dict, Dicti
   }
 }
 
+void DictionaryOperations::WriteDictionarySummaryToLog(const Dictionary& dict) {
+  std::map<ClassId, int> entries_per_class;
+  for (int i = 0; i < dict.size(); i++) {
+    const DictionaryEntry* entry = dict.entry(i);
+    if (entry != nullptr) entries_per_class[entry->token().class_id]++;
```
Comment (MelLain) [inline src/artm/core/dictionary_operations.cc]: new line and { }

File: src/artm/core/dictionary_operations.cc [hunk 4]
```diff
@@ -562,5 +571,18 @@ void DictionaryOperations::StoreIntoDictionaryData(const Dictionary& dict, Dicti
...
+  }
+  std::stringstream ss; ss << "Dictionary name='" << dict.name() << "' contains entries: ";
+  for (auto const& x : entries_per_class)
```
Comment (MelLain) [inline src/artm/core/dictionary_operations.cc]: { }

File: src/artm/core/master_component.cc [hunk 5]
```diff
@@ -380,6 +381,8 @@ void MasterComponent::ExportModel(const ExportModelArgs& args) {
       ::artm::TopicModel external_topic_model;
       PhiMatrixOperations::RetrieveExternalTopicModel(n_wt, get_topic_model_args, &external_topic_model);
       std::string str = external_topic_model.SerializeAsString();
+      if (str.size() >= 2147483647ULL)
```
Comment (MelLain) [inline src/artm/core/master_component.cc]: { }

File: src/artm/core/master_component.cc [hunk 6]
```diff
@@ -474,6 +477,8 @@ void MasterComponent::ExportScoreTracker(const ExportScoreTrackerArgs& args) {
   // We expect here that each ScoreData object has suitable size (< 2GB)
   for (auto& item : instance_->score_tracker()->GetDataUnsafe()) {
     auto str = item->SerializeAsString();
+    if (str.size() >= 2147483647ULL)
```
Comment (MelLain) [inline src/artm/core/master_component.cc]: { }

File: src/artm/score/perplexity.cc [hunk 7]
```diff
@@ -66,7 +66,7 @@ void Perplexity::AppendScore(
       }
     }
     if (class_weight_map.empty()) {
-      LOG(ERROR) << "None of requested classes were presented in model. Score calculation will be skipped";
+      LOG_FIRST_N(ERROR, 100) << "None of requested classes were presented in model. Score calculation will be skipped";
```
Comment (MelLain) [inline src/artm/score/perplexity.cc]: Very optional, but maybe replace 100 with some parameter?

Comment (ofrei) [reply src/artm/score/perplexity.cc]: I'd keep explicit 100 here - there is no need to have consistent number LGO_FIRST_N threshold across library.

File: src/artm/core/dictionary_operations.cc (hunk 1 above)
Comment (ofrei) [reply src/artm/core/dictionary_operations.cc]: Either "to export" or "for exporting". Imho "to export" is more english.

**Pull request merged**